Can I use openpgpjs in a browserified application

I've tried it the naive way, created an `main.js` file

    var openpgp = require('openpgp')

and run the browserify-cli on it

    browserify main.js

Unfortunately it exits with a error:

    Error: Cannot find module './config' from '/home/USER1/Documents/test/node_modules/openpgp/dist'
        at /home/USER1/.npm-packages/lib/node_modules/browserify/node_modules/resolve/lib/async.js:55:21
        at load (/home/USER1/.npm-packages/lib/node_modules/browserify/node_modules/resolve/lib/async.js:69:43)
        at onex (/home/USER1/.npm-packages/lib/node_modules/browserify/node_modules/resolve/lib/async.js:92:31)
        at /home/USER1/.npm-packages/lib/node_modules/browserify/node_modules/resolve/lib/async.js:22:47
        at FSReqWrap.oncomplete (fs.js:117:15)

Same here. I'm using gulp:

```
Error: Cannot find module './config' from '/Users/acasajus/Devel/proj/front/node_modules/openpgp/dist'
    at /Users/acasajus/Devel/proj/front/node_modules/resolve/lib/async.js:55:21
    at load (/Users/acasajus/proj/front/node_modules/resolve/lib/async.js:69:43)
    at onex (/Users/acasajus/proj/front/node_modules/resolve/lib/async.js:92:31)
    at /Users/acasajus/proj/front/node_modules/resolve/lib/async.js:22:47
    at FSReqWrap.oncomplete (fs.js:82:15)
```

--- a/Gruntfile.js
+++ b/Gruntfile.js
@@ -51,7 +51,8 @@ module.exports = function(grunt) {
               ignore: ['*.min.js'],
               presets: ["es2015"]
             }]
-          ]
+          ],
+          plugin: [ 'browserify-derequire' ]
         }
       },
       openpgp_debug: {
@@ -69,7 +70,8 @@ module.exports = function(grunt) {
               ignore: ['*.min.js'],
               presets: ["es2015"]
             }]
-          ]
+          ],
+          plugin: [ 'browserify-derequire' ]
         }
       },
       worker: {

--- a/package.json
+++ b/package.json
@@ -35,6 +35,7 @@
     "asmcrypto-lite": "^1.0.0",
     "babel-preset-es2015": "^6.3.13",
     "babelify": "^7.2.0",
+    "browserify-derequire": "^0.9.4",
     "chai": "~3.5.0",
     "es6-promise": "^3.1.2",
     "grunt": "~0.4.5",

